browser-features/chrome/common/split-view/styles/split-view.css

```
@@ -272,9 +337,21 @@
   > :not(.split-view-panel):not(.floorp-grid-handle),
 #tabbrowser-tabpanels[splitview][split-view-layout="grid-3pane-left-main"]
   > :not(.split-view-panel):not(.floorp-grid-handle),
+#tabbrowser-tabpanels[splitview][split-view-layout="grid-3pane-right-main"]
+  > :not(.split-view-panel):not(.floorp-grid-handle),
```

Stylelint (17.6.0)

[error] Expected complex :not() pseudo-class notation (selector-not-notation)

(selector-not-notation)